src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      return a / b;
```

Comment:
[CODE KNOWLEDGE] - The `divide` method should only take one parameter and divide the current `result` by this parameter. Also, consider handling the case when the parameter is 0 to avoid division by zero.